src/forms/datepicker/DatePicker.tsx

```diff
@@ -17,6 +17,7 @@ export type DatePickerProps = {
     dataqa?: string;
     onChange?: (newValue: string | Date) => void;
     disabled?: boolean;
+    disableFutureDates?: boolean;
```

Comment:
Please add documentation for the below props,
@param {onChange}
@param { disabled}
@param { disableFutureDates}

Reply:
Added the documentation.